addon/components/simple-input.js

```diff
@@ -141,7 +141,7 @@ const extension = {
     }
 
     var result;
-    if (this.get("i18n") && this.get("i18n").exists(key)) { result = this.get("i18n").t(key); }
+    if (this.get("translationService.hasTranslationService") && this.get("translationService.translationService").exists(key)) { result = this.get("translationService.translationService").t(key); }
```

Comment:
I'd prefer if the service encapsulated the `exists` and `t` method. In this case we depend in whole addon on the fact that both `ember-i18n` and `ember-intl` have those methods. I think it'd be much cleaner and safer if we abstracted this knowledge.

As long as both those libraries have both those methods simple delegation would be enough: `exists() { return this.get("translationService").exists(...arguments); }`.